nestor_api/api/api_routes/workflow/workflow.py

```diff
@@ -0,0 +1,60 @@
+"""Define the workflows routes."""
+from http import HTTPStatus
+import traceback
+
+from nestor_api.adapters.git.provider import get_git_provider
+import nestor_api.lib.config as config_lib
+import nestor_api.lib.workflow as workflow_lib
+from nestor_api.utils.logger import Logger
+
+
+def init_workflow(organization, app):
+    """Initialize workflow of an application."""
+    Logger.debug(
+        {"app": app, "organization": organization},
+        "[/api/workflow/init/:org/:app] Workflow initialization started",
+    )
+
+    report = None
+    try:
+        # Retrieve project configuration
+        project_config = config_lib.get_project_config()
```

Comment:
Be careful here, you should first create a copy of the configuration to ensure that no concurrent request updates the config while you use it.
Have a look at what is made [in the build api](https://github.com/ChauffeurPrive/nestor-api/blob/master/nestor_api/api/api_routes/builds/build_app.py#L22-L24)

It should look like this :
```python
config_dir = config_lib.create_temporary_config_copy()
config_lib.change_environment(Configuration.get_config_default_branch(), config_dir)
app_config = config_lib.get_app_config()
```

This makes me raise, once again, a question you had before :
- what should the default branch be ? how should we determine it ?

Reply:
But I don't want the app config, I want the project config. The feedback is still valid as there could be some concurrent request for updates but it should not move that often. Anyway, I will create a local copy, for the project config so it will indeed be safer.